sequencescape.code-workspace

```diff
@@ -4,5 +4,10 @@
 			"path": "."
 		}
 	],
-	"settings": {}
-}
\ No newline at end of file
+	"settings": {
+		"cSpell.words": [
+			"barcode",
```

Comment:
it didn't know the word 'barcode'?

Reply:
Huh, hadn't realised I'd added this to the workspace. Oddly was experimenting with it in a different branch.